modules/core/src/image/vpFont.cpp

```diff
@@ -2172,6 +2172,7 @@ class vpFont::Impl
 
   void LoadTTF(const std::string & filename)
   {
+#if 0 // disable for compatibility with old platform (Ubuntu 12.04)
     std::ifstream fontFile(filename.c_str(), std::ios::binary |std::ios::ate);
```

Comment:
I suggest to replace `#if 0` by `#if (VISP_CXX_STANDARD > VISP_CXX_STANDARD_98)`